setup.py

```diff
@@ -27,10 +27,11 @@
     keywords='Telegram messaging database',
     packages=find_packages(),
     install_requires=[
-        'tqdm', 'telethon-aio', 'appdirs',
+        'tqdm', 'telethon-aio', 'appdirs', 'PySocks',
         'async_generator'  # Python 3.5 async gen support
     ],
     scripts=['bin/telegram-export'],
+    test_suite='telegram_export.tests',
```

Comment:
Nice, may be better suited as a separate pull request perhaps?

**Edit**: nevermind, you added tests for the proxy, that's fine.